### ISSUE TITLE
Missing <span class="caret"></span> in dropdown select?

### ISSUE THREAD

**Issue by noru** [kzima/vuestrap-base-components#28]
Hi,

I found 'caret' prop is not working for dropdown select(it works well with dropdown).

After checking the code, I found **dropdown-select.html** is missing `<span class="caret"></span>`.

Is it the problem cause?

Thanks!

**Comment by tvld** [comment]
I think the repo ownder decided to drop support for this project. We use it to great satisfaction in our websites. Wondering if there is a guru (with more knowledge than we have) to fork and adopt this ?

**Comment by taylorzane** [comment]
/cc @noru @tvld 

@noru Were you looking to create a pull request?  If not, I can definitely fix this issue for you. But that would appear to be the case. The Vue component seems to reference a computed prop named `dropdownToggle` which checks to see if `caret` is `true`.

I'm using his set of repositories pretty extensively. Kris has been pretty good about publishing changes to npm. If the time comes where Kris isn't willing to push changes to npm, I'll probably end up forking it and maintaining it.

**Comment by noru** [comment]
@tvld Sorry to hear that
@taylorzane Thanks for your reply. I haven't really dug into this issue (or this project) yet, and it isn't so important one. But I do hope such useful project to keep an active state. It will be great if you intend to maintain it, and I guess people would be glad to contribute (I sure would :) )

**Comment by tvld** [comment]
It seems there is positive energy building again ! Let's keep it going ))

**Comment by noru** [comment]
@taylorzane hi, sorry to bother you again. I bumped into a few other defects in my project. They are trivials but I'd really like them to be fixed in a timely manner. Also, I found there are important features still missing in existing components.
Since you are willing to fork it and maintain it (I am too), shall we make a move? Maybe by asking Kris if it is ok to hand over the package first?

**Comment by tvld** [comment]
@noru As for myself, yesterday I investigated Bulma, made a first small site and contemplating to leave Bootstrap behind as for the maintenance reason. I dont like BS v3 and I did not see any other BS v4 x vue implementations. So I think you are suggesting the right thing to do, given [feedback of Kris](https://github.com/kzima/vuestrap-base-components/issues/21) some time ago: _" I just have no time to maintain it anymore and I hope other people will contribute to this"_

**Comment by noru** [comment]
@tvld Thanks for your response, let's keep it going then. Vue2 is coming out now, it's also great if this project can support that as well. 
As soon as @taylorzane respond and consent, let's take another stop :)

**Comment by tvld** [comment]
@noru ok then. Bit confused by the way how it is possible that both vue and BS are  so big now and so few people seem to bother about this combination... well... maybe still under the radar? ))

**Comment by tvld** [comment]
I am still in limbo. 
I really dont like how theming is done in this "vuestrap'. Webpack preloader or some hacks I used before.

What I like about [Bulma](http://bulma.io/), see also  [Vue Admin](https://github.com/fundon/vue-admin) that basically you just override a few sass variables and you good to go. Nice and clean. After the tests were ok last night...  I leaning towards sticking around there. It's a long term decision so a hard one, that I admit...

**Comment by noru** [comment]
@tvld, thanks for your information. I think we share the same struggling here, and there probably some underlying reason that Vue and BS don't go well together (maybe Kris knows better about it so decided to drop it?). Anyway, I'm going to check out other options like Bulma you suggested.

**Comment by tvld** [comment]
@noru  Yes, you might be right . BS still relies heavily on jQuery... and vue replaced jQuery, for me at least. Bulma is (s)css  only, which makes it less of an headache to integrate. It's bit of a learning curve but not major. So far I like it, first app already converted  ... :)

**Comment by kzima** [comment]
@noru, @tvld: I was like you guys searching for the best combination of js framework and some well tested css library to build websites quicker than before. It was fun to work with vue js and Bootstrap 4. This combination however is perfect only for the web apps or wordpress/laravel hybrid type of apps. Today I work on single page apps which are often compiled to native apps and there are components built for that already, like this for example: http://www.material-ui.com/#/components/app-bar. 

So, if you want to combine vue 2.0 with B4, I think it's a good idea. There is few tips I can give though. Write docs in markdown and do not bother with html docs, unless there are more collaborators. I remember docs consumed half of my dev time :( 
I would also make simple default components that can be easily extended/overridden with js and css. If I kept my components simple with just one color variant I would probably save a lot of time. The thing is to build as much traction as possible in the shortest time! And by the time you finish all your components, you have to be ready to implement vue 3.0 :) and so on!  Endless development, hehe. Good luck!
